bins/revm-test/src/bin/burntpix/static_data.rs

```diff
@@ -0,0 +1,19 @@
+use revm::primitives::{address, bytes, Address, Bytes};
+
+pub const BURNTPIX_MAIN_ADDRESS: Address = address!("49206861766520746f6f206d7563682074696d65");
+pub const BURNTPIX_ADDRESS_ONE: Address = address!("0a743ba7304efcc9e384ece9be7631e2470e401e");
+pub const BURNTPIX_ADDRESS_TWO: Address = address!("c917e98213a05d271adc5d93d2fee6c1f1006f75");
+pub const BURNTPIX_ADDRESS_THREE: Address = address!("f529c70db0800449ebd81fbc6e4221523a989f05");
+
+pub const STORAGE_ZERO: &str = "0x000000000000000000000000f529c70db0800449ebd81fbc6e4221523a989f05";
+pub const STORAGE_ONE: &str = "0x0000000000000000000000000a743ba7304efcc9e384ece9be7631e2470e401e";
+pub const STORAGE_TWO: &str = "0x000000000000000000000000c917e98213a05d271adc5d93d2fee6c1f1006f75";
```

Comment:
You can probably use https://docs.rs/alloy-primitives/0.6.0/alloy_primitives/macro.uint.html to get const `U256`

Reply:
Or maybe https://docs.rs/alloy-primitives/0.6.0/alloy_primitives/macro.fixed_bytes.html